src/Models/Rate.php

```diff
@@ -0,0 +1,182 @@
+<?php namespace Tipoff\EscapeRoom\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Rate extends Model
+{
+    use HasFactory;
+
+    protected $guarded = ['id'];
+
+    protected $casts = [];
+
+    protected static function boot()
+    {
+        parent::boot();
+
+        static::creating(function ($rate) {
+            if (auth()->check()) {
+                $rate->creator_id = auth()->id();
+            }
+        });
+
+        static::saving(function ($rate) {
+            if (auth()->check()) {
+                $rate->updater_id = auth()->id();
+            }
```

Comment:
I'm going to pull this out in all the packages to a HasUpdater trait in the support package.